src/auth/auth-api-request.ts

```diff
@@ -1644,7 +1644,7 @@ export class TenantAwareAuthRequestHandler extends AbstractAuthRequestHandler {
    * @return {AuthResourceUrlBuilder} A new project config resource URL builder instance.
    */
   protected newProjectConfigUrlBuilder(): AuthResourceUrlBuilder {
-    return new TenantAwareAuthResourceUrlBuilder(this.projectId, 'v2beta1', this.tenantId);
+    return new TenantAwareAuthResourceUrlBuilder(this.projectId, 'admin/v2', this.tenantId);
```

Comment:
It seems they plan to remove the `admin/` prefix as it is not one platform compliant (it will break the API discovery doc).
They are working on it now.

Reply:
Removed the prefix from the calls.